tests/data/quartznet_spkr_test.yaml

```diff
@@ -0,0 +1,81 @@
+model: "GramVoxNet"
```

Comment:
Is this file any different from your configs inside of examples?

Reply:
Yeah smaller emb size